src/app/config.json

```diff
@@ -5,8 +5,25 @@
     "results": "result",
   },
   "syncOptions" : {
-    "sync_frequency" : 5,
-    "storage_strategy": "dom",
-    "do_console_log": false
+    "global":{
+      "sync_frequency" : 1,
+      "storage_strategy": "dom",
+      "do_console_log": false
+    },
+    "workorders":{
+      "sync_frequency" : 2,
```

Comment:
2 seconds is overkill even for demo app. This will drain battery a lot and with new sync it can take more than one sync loop to get changes back. I think that 10 seconds should be sensible default.